### PR TITLE
perf: update readme and package.json for add sso in bot

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

**Review by IvanJobs — DISMISSED** [review]
lgtm

File: packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ These folder contains three files as reference for sso implementation:
     ```
     with:
     ```
-    "build": "tsc --build && copyfiles public/*.html lib/",
+    "build": "tsc --build && copyfiles src/public/*.html lib/",
```
Comment (swatDong) [inline packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:51]: Does this conflict with #4963 ?

Comment (KennethBWSong) [reply packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:51]: Yes...... I will align with #4963 , thx!

**Review by swatDong — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
lgtm

**Pull request merged**